deploy/azure/README.md

```diff
@@ -0,0 +1,252 @@
+and helm installed (see below for instructions on installing
+helm). Note the following requirements:
+
+* The helm chart deploys Prometheus and Grafana to collect and display Triton metrics. To use this helm chart you must install Prpmetheus and Grafana in your cluster as described below and your cluster must contain sufficient CPU resourses to support these services.
+
+* If you want Triton Server to use GPUs for inferencing, your cluster
+must be configured to contain the desired number of GPU nodes (Azure NC series VMs recommended)
+with support for the NVIDIA driver and CUDA version required by the version
+of the inference server you are using.
+
+The steps below describe how to set-up a model repository, use helm to
+launch the inference server, and then send inference requests to the
+running server. You can access a Grafana endpoint to see real-time
+metrics reported by the inference server.
+
+## Installing Helm
+
+If you do not already have Helm installed,
+executing the following steps from the [official helm install
+guide](https://helm.sh/docs/intro/install/) will
+give you a quick setup.
+
+```sh
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
+```
+
+## Model Repository
+
+If you already have a model repository you may use that with this helm
+chart. If you do not have a model repository, you can checkout a local
+copy of the inference server source repository to create an example
+model repository:
+
+```sh
+$ git clone https://github.com/triton-inference-server/server.git
+```
+
+Triton Server needs a repository of models that it will make available
+for inferencing. For this example you will place the model repository
+in an Azure Storage Container in an Azure Storage Account. Replace the value of `$AZURE_STORAGE_ACCOUNT` with the name of an Azure Storage Account you have access to.
+
+```sh
+$ AZURE_STORAGE_ACCOUNT=mystorageaccount
+$ AZURE_STORAGE_CONTAINER=triton-inference-server-repository
+
+$ az storage container create --name $AZURE_STORAGE_CONTAINER --account-name $AZURE_STORAGE_ACCOUNT
+```
+
+Following the [QuickStart](../../docs/quickstart.md) download the
+example model repository to your system and copy it into the Azure Storage Container.
+
+```sh
+$ cd ./docs/examples
+$ az storage blob directory upload -c $AZURE_STORAGE_CONTAINER -d . -s model_repository --account-name $AZURE_STORAGE_ACCOUNT --recursive
+```
+
+### Azure Model Repository
+To load models from the Blob Storage Container, you need to convert the following Azure credentials to base64 format and add them to the Helm Chart's [values.yaml](./values.yaml)
+
+For `AZURE_STORAGE_ACCOUNT_NAME`:
+```sh
+$ echo -n $AZURE_STORAGE_ACCOUNT | base64
+```
+
+For `AZURE_STORAGE_KEY`:
+```sh
+$ AZURE_STORAGE_KEY=`az storage account keys list -n $AZURE_STORAGE_ACCOUNT --query "[0].value"`
+
+$ echo -n $AZURE_STORAGE_KEY | base64
```

Comment:
$ echo -n $AZURE_STORAGE_KEY | base64 -w 0

Add `-w 0` to `base64` - My key was being wrapped to a new line and causing issues.